client/lib/plans/features-list.tsx

```diff
@@ -1925,6 +1931,30 @@ export const FEATURES_LIST: FeatureList = {
 				'Get 30 social shares per month to promote your posts on Facebook, Twitter, Tumblr, and more.'
 			),
 	},
+	[ FEATURE_SHARES_SOCIAL_MEDIA_JP ]: {
+		getSlug: () => FEATURE_SHARES_SOCIAL_MEDIA_JP,
+		getTitle: () => i18n.translate( 'Shares on social media' ),
+		getDescription: () =>
+			i18n.translate(
+				'Automatically share your latest post on Facebook, Twitter, Tumblr, and more.'
+			),
+		getConditionalTitle: ( planSlug ) => {
+			if ( ! planSlug ) {
+				return '';
+			}
+			if ( isPersonalPlan( planSlug ) || isFreePlan( planSlug ) ) {
+				return i18n.translate( '%d shares per month', { args: [ 30 ] } );
+			}
+			if (
+				isPremiumPlan( planSlug ) ||
+				isBusinessPlan( planSlug ) ||
+				isEcommercePlan( planSlug )
+			) {
+				return i18n.translate( 'Unlimited shares' );
+			}
+			return '';
```

Comment:
Not a fan of bringing in plan functions in the feature list, but I could not find an alternate way to do this in the current plans framework. Open to suggestions!

Reply:
I agree. Until the day our data definition allows us to write only by some simple slugs, I think this is the best we can do. 🤷🏼